.github/actions/bumpVersion/action.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Bump npm version'
+description: 'Increase the npm build version of the ReactNativeChat application'
+inputs:
+    githubToken:
```

Comment:
It's not 100% convention, but most actions use `GITHUB_TOKEN`, so I think we should match that here.